sdk/core/System.Net.ClientModel/tests/clients/OpenAIClientTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using NUnit.Framework;
+using OpenAI;
+
+namespace System.Net.ClientModel.Tests;
+
+public class OpenAIClientTests
+{
+    [Test]
+    public void TestClientSync()
```

Comment:
This test will pass today if you comment out the final lines of the test function.